### ISSUE TITLE
TypeError: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by bennymeier** [pytesNET/tail.select#32]
I got a list like this:
- Entry 1
- Entry 2
- Entry 3

I used the remove() method like this:
tail.options.remove("Entry 1", "#", true);
tail.options.remove("Entry 2", "#", true);
tail.options.remove("Entry 3", "#", true);

There is no entry left in the list and this error comes up.
`Uncaught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at p.options.walker (tail.select-full.min.js?e367:formatted:884)
    at p.options.walker (tail.select-full.min.js?e367:formatted:884)
    at p.query (tail.select-full.min.js?e367:formatted:884)
    at p.options.remove (tail.select-full.min.js?e367:formatted:884)
    at UpDistributionControl.removeValue (UpDistributionControl.ts?fd55:84)
    at <anonymous>:1:7`

**Comment by SamBrishes** [comment]
Hellow,

I'm currently not really sure why this error occurs, it must be something between the `tailOptions` class and the `.query()` method, but it is also really late... so... **I found a solution** which fixes this issue, until the new major release 0.6.0 will handle this.

Just set the third option on the `.options.remove()` method to `false` (which disables the rebuilding) and rebuild the `tail.select` instance on your own (as shown [on this jsFiddle](https://jsfiddle.net/SamBrishes/fuo5ahcv/4/)):

```javascript
let select = tail.select("#my-select");

select.options.remove("Entry 1", "#", false);
select.options.remove("Entry 2", "#", false);
select.options.remove("Entry 3", "#", false);

select.reload();
```

Thanks for reporting.

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Hellow,

this issue has been fixed with 0.5.7, the new version is also already available!

As a small side note: The third parameter of the `.remove()` method allows to re-render the optionslist. If you want to remove multiple elements / options at once, use `false` as third argument **except** on the last method call! So the re-rendering of the optionslist doesn't happen on each single removed option, but only after all desired options has been removed.

Sincerely,
Sam.
